tests/baselines/reference/doubleUnderscoreSymbolsAndEmit.errors.txt

```diff
@@ -0,0 +1,60 @@
+tests/cases/compiler/index.tsx(33,1): error TS2308: Module "./b" has already exported a member named '__foo'. Consider explicitly re-exporting to resolve the ambiguity.

```

Comment:
I didn't expect a test named doubleUnderscoreSymbolsAndEmit to have errors. Is this expected?